app/workers/send_request_events_to_bigquery.rb

```
@@ -0,0 +1,19 @@
+class SendRequestEventsToBigquery
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 3, queue: :low_priority
+
+  TABLE_NAME = 'apply_request_events'.freeze
+
+  def perform(request_event_json)
+    big_query_project_id = ENV['BIG_QUERY_PROJECT_ID']
+    big_query_dataset = ENV['BIG_QUERY_DATASET']
```

Should these use `fetch`? Good to fail early in case we misconfigured